apps/stripe/src/app/api/webhooks/stripe/stripe-webhook-response.ts

```diff
@@ -4,19 +4,23 @@ export class StripeWebhookSuccessResponse {
   readonly responseStatusCode = 200;
 
   getResponse() {
-    // todo What should be the message
-    return new Response("OK", {
+    return new Response("Ok", {
       status: this.responseStatusCode,
     });
   }
 }
 
-/**
- * Probably Stripe doesn't retry even if webhook failed.
- *
- * TODO: Check how to handle retrying webhooks
- */
-export class StripeWebhookErrorResponse {
+export class StripeWebhookNonRetryableErrorResponse {
+  readonly responseStatusCode = 400;
+
+  getResponse() {
+    return new Response("Malformed request", {
```

Comment:
This is not true anymore - it's non retryable, but it doesn't mean it's always malformed.

We can return non-retryable when we fail to fetch config.

So probably we should have some base class (abstract maybe) + specific error like `MalformedRequest extends StripeWebhookNonRetryableErrorResponse` or `MissingConfig extends StripeWebhookNonRetryableErrorResponse`